src/python/foglamp/logger.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Syslog logger """
+
+
+import logging
+from logging import handlers
+
+__author__ = "Praveen Garg"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+def setup(name):
+    """ Configures the log mechanism """
+
+    try:
+        handler = handlers.SysLogHandler(address='/dev/log')  # OSX: /var/run/syslog, Ubuntu: /dev/log
```

Comment:
Yes, I mean, let's not refer to OSX because FogLAMP does not meant to run on OSX. This only confuses a developer who may get the code. By reading the comment, the developer may think that FogLAMP works on OSX.